xAPI.md

```diff
@@ -3452,6 +3459,8 @@ the latest minor and patch version the LRS conforms to, for each major version.
     * For version 1.0.0 of this specification, this means that "1.0.0" MUST be included;
     "0.9" and "0.95" MAY be included. (For the purposes of this requirement, "0.9" and "0.95"
     are considered major versions.)
+* Additional properties MUST NOT be added to this object outside of extensions and each 
+property MUST occur only once.  
```

Comment:
MUST for multiplicity as above. Here I'm confident with a MUST for additional properties too as we explicitly say on [line 3457](https://github.com/adlnet/xAPI-Spec/pull/580/files#diff-68cd6732e400b6e64376effdb84b970eR3457) ```An LRS MUST return the JSON document described above```.